src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3937,7 +3937,8 @@ internal void UpdateTypes(ExecutionContext context, bool updateOnly)
 
             if (errors.Count > 0)
             {
-                var allErrors = new StringBuilder('\n');
+                // Note: don't use '\n' ('char' convert implicitly to 'ushort' and 'int' and initialize 'StringBuilder' size)
```

Comment:
@lzybkr  Thanks for great comment!
Here we cannot use a constant, the capacity is ~(1+ 2 * errors.Count).

Do it make sense to open CoreFX issue to add new constructor for StringBuilder ?
`StringBuilder new(char value)`